apps/evm_family/evm_priv.h

```diff
@@ -25,6 +25,7 @@
 
 #define EVM_TRANSACTION_SIZE_CAP 20480
 
+#define MAX_MSG_DATA_SIZE 5120
```

Comment:
Please unify the value of this macro with eth sign txn max cap

Reply:
Added a todo in evm_sign_msg.c not unifying the size as of now as there's an issue of double allocation for typed data

Reply:
Also the size will not be supported by personal sign display

Reply:
add these points to the source doxygen comment. and make the acceptable size to something that would not break the device. also, add any possible solution to this size problem for future reference.

Reply:
Will add this in the typed data PR.
For this PR the solution is to keep max size at 5kb(same as previous) which will although break considering the LVGL buffer issue.
For the LVGL issue I've added a todo before `core_scroll_page` call.

Reply:
added a todo

Reply:
updated the size to avoid failure